Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -100,11 +100,17 @@ CGFontRef CGFontRetain(CGFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 CGFontRef CGFontCreateWithDataProvider(CGDataProviderRef cgDataProvider) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    size_t memSize = sizeof(struct __CGFont) - sizeof(CFRuntimeBase);
+    CGFontRef ret = static_cast<CGFontRef>(_CFRuntimeCreateInstance(kCFAllocatorDefault, CGFontGetTypeID(), memSize, NULL));
+    CFAutorelease(ret);
+    struct __CGFont* mutableRet = const_cast<struct __CGFont*>(ret);
```

Comment:
Have concerns about losing Hash, Description, Equal functions from above. Need to talk to @DHowett-MSFT about those wrt CppBase. Noted for #1212

---
In reply to: [88146106](https://github.com/Microsoft/WinObjC/pull/1386#discussion_r88146106) [](ancestors = 88146106)